### PR TITLE
Implement classes count reporting.

### PR DESCRIPTION
**Pull request by stefan-nsk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ in `app/build.gradle`:
 dexcount {
     format = "list"
     includeClasses = false
+    includeClassCount = true
```
Comment (benjamin-bader) [inline README.md]: Nit: this should be `false` to match the default value in `DexMethodCountExtension`

**Comment by benjamin-bader** [comment]
Hey, thanks for your contribution!  Solid work; just a few more tests needed, then this will be good to go!

**Comment by stefan-nsk** [comment]
done

**Comment by benjamin-bader** [comment]
Thanks!

thumbs up

**Comment by stefan-nsk** [comment]
Thanks! What are your plans to release a new version?

**Comment by benjamin-bader** [comment]
No definite schedule.  I'd like to fix compatibility with Android Studio 3.0's build tools first (#161) but haven't yet figured out how.

**Pull request merged**